gradle.properties

```diff
@@ -3,7 +3,7 @@ version=2.0.0.999
 #kotlinVersion=1.5.20
 kotlinVersion=1.5.30-M1
 kotlinSerializationVersion=1.2.1
-coroutinesVersion=1.5.0
+coroutinesVersion=1.5.1-SNAPSHOT
```

Comment:
I just remembered this relies on a custom build of kotlinx.coroutines with Arm32Hfp enabled.  Probably can't merge this unless that gets enabled upstream.

Reply:
I'm going to merge it and disable it with a flag or something, so the code is already here and we can enable it when this situation changes.

Did you do something special to compile kotlinx.coroutines & atomicfu on the PI?

Reply:
For kotlinx.coroutines I needed to disable tests on the arm target: https://gist.github.com/ObsidianX/dcfb862a6f41fad598053eef50679374

Reply:
Because some of the tests failed, or something else?

Reply:
Posted a PR with a better solution: https://github.com/Kotlin/kotlinx.coroutines/pull/2841